core/arch/arm/include/ffa.h

```diff
@@ -278,13 +278,32 @@ struct ffa_mem_access_perm {
 	uint8_t flags;
 };
 
-/* Endpoint memory access descriptor */
-struct ffa_mem_access {
+/* Endpoint memory access descriptor up to version 1.1 */
+struct ffa_mem_access_1_1 {
```

Comment:
I find the naming slightly misleading. Here, `_1_1` means "up to version 1.1", while in `ffa_mem_access_1_2` below is means "from version 1.2". Should this be `ffa_mem_access_1_0` instead? It would make `struct ffa_mem_transaction_1_0` contain a `struct ffa_mem_access_1_0` which looks more logical than it having a `struct ffa_mem_access_1_1`.